local-cli/server/middleware/getDevToolsMiddleware.js

```diff
@@ -24,7 +25,42 @@ function getChromeAppName() {
   }
 }
 
-module.exports = function(options, isDebuggerConnected) {
+function launchChromeDevTools(options) {
+    var debuggerURL = 'http://localhost:' + options.port + '/debugger-ui';
+    console.log('Launching Dev Tools...');
+    opn(debuggerURL, {app: [getChromeAppName()]}, function(err) {
+        if (err) {
+            console.error('Google Chrome exited with error:', err);
+        }
+    });
+}
+
+function escapePath(path) {
+  return path.indexOf(' ') > -1
+    ? '"' + path + '"' // " Can escape paths with spaces in OS X, Windows, and *nix
+    : path;
+}
+
+function launchDevTools(options, isChromeConnected) {
+    // Explicit config always wins
+    var customDebugger = process.env.REACT_DEBUGGER;
+    if (customDebugger) {
+        var projects = options.projectRoots.map(escapePath).join(' ');
+        var command = customDebugger + ' ' + projects;
+        console.log('Starting custom debugger by executing: ' + command);
+        child_process.exec(command, function (error, stdout, stderr) {
```

Comment:
child_process.exec uses a buffer of at max 200 kb by default, so I think this should be okay. We are planning on launching a small node process which will launch the real debugger in our case, so the 200 kb limit won’t be an issue. Using exec also makes it easier to put a command in the environment variable, and execute it directly with exec. We'd have to transform the environment variable into two values, one for the command and one for the arguments to make it work with spawn, or we’d be limited to just call and executable program without any custom parameters.